asmcomp/amd64/emit.mlp

```diff
@@ -33,6 +33,7 @@ module String = Misc.Stdlib.String
    emit.mlp files for certain other targets; the reference here ensures
    that when releases are being prepared the .depend files are correct
    for all targets. *)
+[@@@ocaml.warning "-66"]
```

Comment:
Was that required ?

Reply:
It appears to be required with the dune build, because I think it sets different warning levels.  (I also think it is unfortunate that `open!` is falling foul of warning 66.)